src/main/resources/languages/en/messages.yml

```diff
@@ -347,6 +347,7 @@ smithing_table:
 
 villagers:
   no-trading: '&4You cannot trade Slimefun items with Villagers!'
+  no-trading-new: '&4You cannot trade this Slimefun item with Villagers or Wandering Traders!'
```

Comment:
Maybe name this no-trading-individual ? just an idea

Reply:
i really need a better name here...

Reply:
Maybe no-trading-singular, or what Justin suggested: new sounds weird but anything else will work since there is an explanation literally next to it

Reply:
The reason for adding a new message is that changes to the old message will not be deployed to existing servers or Crowdin, so I added a new message.
The old message will not be used anymore, not sure if `singular` is a proper name here.

Reply:
I see the reasoning, not a fan but given the alternative is rewriting the whole message system, I can follow the logic. 

While we’re here, it doesn’t really need to mention Villager or Wandering traders does it, given the message is given during the context of trading the player could just be told `You cannot trade using this Slimefun Item` or something to that effect? That was when mojang adds a… SoitheringMerchant in the future we don’t need to worry about further ‘new’ versions of this same message?

Reply:
> The reason for adding a new message is that changes to the old message will not be deployed to existing servers or Crowdin, so I added a new message.
> 
> The old message will not be used anymore, not sure if `singular` is a proper name here.

I think I explained myself badly: I understand we won't be using the other one anymore but it's going to stay there for now (probably for a long time actually) so we need a way to distinguish the one referring to Slimefun Items as a whole and the one referring to a single one - hence why I was thinking either singular or individual would do.

I don't think perfect naming is needed, just one that is good enough: the error message it represents sits right next to it so the exact meaning can be deduced by context, otherwise we would need "no-trading-is-allowed-for-this-slimefun-item" or something which looks even worse in my opinion, way too many details.

Essentially, if this is the only thing keeping the PR open once the other requested change is addressed, I don't think we should spend too much time on it when it makes almost no difference overall